proto/schema/v1/query.proto

```diff
@@ -40,7 +41,11 @@ message QuerySchemaRequest {
 }
 
 message QuerySchemaResponse {
-  WhatIs schema = 1;
+  // Schema Repersentation
+  WhatIs whatIs = 1;
```

Comment:
I think it's unnecessary to include both of these. Just the definition should be fine

Reply:
What if we need to verify the schema is registered for current app? And situation where App B wants to use schemas from App A? I feel like keeping WhatIs will help alleviate potential unaccounted for issues.

Reply:
> I think it's unnecessary to include both of these. Just the definition should be fine. Ill separate into a separate query method and proto for the `WhatIs`